File: Design and Programming Environments/Iteration 3/TicketManager/Networking/src/main/java/ObjectProtocol/GetAllTicketsResponse.java
package ObjectProtocol;

import DTOPackage.TicketDTO;

public class GetAllTicketsResponse implements Response {
    private TicketDTO[] ticketDTO;

    public GetAllTicketsResponse(TicketDTO[] ticketDTO) {
        this.ticketDTO = ticketDTO;
    }

    public TicketDTO[] getTicketDTO() {
        return ticketDTO;
    }
}

File: Second year/Design and Programming Environments/TicketManagerCSharp/Networking/Responses/GetAllTicketsResponse.cs
using Networking.DTOS;
using System;
using System.Collections.Generic;

namespace Networking.Responses
{
    [Serializable]
    public class GetAllTicketsResponse : Response
    {
        private IList<TicketDTO> tickets;

        public GetAllTicketsResponse(IList<TicketDTO> tickets)
        {
            this.tickets = tickets;
        }

        public IList<TicketDTO> Tickets { get => tickets; set => tickets = value; }
    }
}